gcampus/core/templates/gcampuscore/components/bach_index.html

```diff
@@ -0,0 +1,114 @@
+                        {% translate "N/A" context "no value provided (water name, comment, etc)" as not_any %}
+                    {% endif %}
+                </div>
+            </div>
+            <div class="col-5">
+                <div class="text-secondary fw-light">
+                    {% translate "Validity" %}
+                </div>
+                <div class="fw-bold">
+                    {% widthratio measurement.bach_index.validity 1 100 %}%
+                </div>
+            </div>
+            <div class="col-4">
+                <div class="text-secondary fw-light">
+                    {% translate "Parameters" %}
+                </div>
+                <div class="indicator-list">
+                    <div class="indicator-container">
+                        <i class="circle-icon indicator-icon no3 {% has_parameter_type measurement "no3" %}"
+                           data-bs-toggle="tooltip" title="{% translate "Nitrate" %}"></i>
+                    </div>
+                    <div class="indicator-container">
+                        <i class="circle-icon indicator-icon nh4 {% has_parameter_type measurement "nh4" %}"
+                           data-bs-toggle="tooltip" title="{% translate "Ammonium" %}"></i>
+                    </div>
+                    <div class="indicator-container">
+                        <i class="circle-icon indicator-icon po4 {% has_parameter_type measurement "po4" %}"
+                           data-bs-toggle="tooltip" title="{% translate "Phosphate" %}"></i>
+                    </div>
+                    <div class="indicator-container">
+                        <i class="circle-icon indicator-icon ph {% has_parameter_type measurement "ph" %}"
+                           data-bs-toggle="tooltip" title="{% translate "pH" %}"></i>
+                    </div>
+                    <div class="indicator-container">
+                        <i class="circle-icon indicator-icon conductivity {% has_parameter_type measurement "conductivity" %}"
+                           data-bs-toggle="tooltip" title="{% translate "Conductivity" %}"></i>
+                    </div>
+                    <div class="indicator-container">
+                        <i class="circle-icon indicator-icon temp {% has_parameter_type measurement "temp" %}"
+                           data-bs-toggle="tooltip" title="{% translate "Temperature" %}"></i>
+                    </div>
+                    <div class="indicator-container">
+                        <i class="circle-icon indicator-icon o2 {% has_parameter_type measurement "o2" %}"
+                           data-bs-toggle="tooltip" title="{% translate "Oxygen" %}"></i>
+                    </div>
+                    <div class="indicator-container">
+                        <i class="circle-icon indicator-icon bsb5 {% has_parameter_type measurement "bsb5" %}"
+                           data-bs-toggle="tooltip" title="{% translate "BSB5" %}"></i>
```

Comment:
Can you provide context to this translation? This makes it easier to translate the string later.